src/views/pages/components/macro.njk

```diff
@@ -16,8 +16,8 @@
   </div>
 {% endmacro %}
 {% macro multiSideBar(sideBars) %}
-  <div class="govuk-grid-column-one-third  govuk-!-padding-top-4 govuk-body">
-    <hr class="sideBarColor"/>
+  <div class="govuk-grid-column-one-third  govuk-!-padding-top-0 govuk-body">
+    <hr style="background-color:#1d70b8; height:2px; border:none;">
```

Comment:
does it affect country question in any way?

Reply:
Required to match the style of the prototype.

Reply:
Same as below with screenshots of prototype and the app itself - it changed the designs and is no longer matching prototype for country page.

Reply:
As discussed resolving now